keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/credential/Ssha256PasswordHashProvider.java

```diff
@@ -0,0 +1,87 @@
+package io.cloudtrust.keycloak.credential;
+
+import org.keycloak.common.util.Base64;
+import org.keycloak.credential.hash.PasswordHashProvider;
+import org.keycloak.models.PasswordPolicy;
+import org.keycloak.models.credential.PasswordCredentialModel;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Arrays;
+
+/**
+ * Implementation of the Salted SHA256 password hash algorithm.
+ */
+public class Ssha256PasswordHashProvider implements PasswordHashProvider {
+    private final String algorithm = "ssha256";
```

Comment:
Why not relying on the ID defined in the factory ? are those two values really different in meaning ?

Reply:
No they aren't. I modified it.